js/newtab.js

```diff
@@ -159,6 +159,10 @@ function getImage(info) {
 		// Make a new request to get list of team media from TBA
 		var mediaReq = new XMLHttpRequest();
 
+		// Get month and year to determine what year the photo is from
+		var month = new Date().getMonth();
+		var year = new Date().getFullYear();
```

Comment:
You're still making two `Date` objects. You can do
```js
var date = new Date();
```
and then later on:
```js
date.getMonth()
```
etc.